mgclient_cpp/include/mgclient.hpp

```diff
@@ -57,6 +57,7 @@ class Client {
   struct Params {
     std::string host = "127.0.0.1";
     uint16_t port = 7687;
+    std::string scheme = "basic";
```

Comment:
I think default scheme should be none https://neo4j.com/docs/bolt/current/bolt/message/#messages-hello

Reply:
Changed to `none`